lib/iris/fileformats/netcdf.py

```diff
@@ -1831,6 +1848,54 @@ def _create_cf_data_variable(self, cube, dimension_names, local_keys=None,
             The newly created CF-netCDF data variable.
 
         """
+        if 'packing' in kwargs:
+            packing = kwargs.pop('packing')
+            if packing:
+                scale_factor = None
+                add_offset = None
+                fill_value = None
+                masked = isinstance(cube.data, np.ma.core.MaskedArray)
```

Comment:
Can you use `ma.` instead of `np.ma.` please. There is already an `import numpy.ma as ma` in the imports section.